www/includes/easyparliament/templates/html/search/results.php

```diff
@@ -17,104 +18,104 @@
+                          <?php } else if ( $sort_order == 'oldest' ) { ?>
+                            <li>Sort by <a href="<?= $urls['relevance'] ?>">relevance</a></li>
+                            <li>Sorted by date: <a href="<?= $urls['newest'] ?>">newest</a> / oldest</li>
+                          <?php } else { ?>
+                            <li>Sort by <a href="<?= $urls['relevance'] ?>">relevance</a></li>
+                            <li>Sorted by date: newest / <a href="<?= $urls['oldest'] ?>">oldest</a></li>
+                          <?php } ?>
+                            <li><a href="<?= $urls['by-person'] ?>">Group by person</a></li>
+                        </ul>
+                      <?php } ?>
+
+                      <?php foreach ( $rows as $result ) { ?>
+                        <div class="search-result search-result--generic">
+                            <h3 class="search-result__title"><a href="<?= $result['listurl'] ?>"><?= $result['parent']['body'] ?></a> (<?= format_date($result['hdate'], SHORTDATEFORMAT) ?>)</h3>
+                            <p class="search-result__description"><?= isset($result['speaker']) ? $result['speaker']['name'] . ': ' : '' ?><?= $result['extract'] ?></p>
+                        </div>
+                      <?php } ?>
 
                     <hr>
 
-                    <?php if ( $pagination_links ) { ?>
-                    <div class="search-result-pagination">
-                        <?php if ( isset($pagination_links['prev']) ) { ?>
-                        <a href="<?= $pagination_links['firstpage']['url'] ?>" title="First page">&lt;&lt;</a>
-                        <a href="<?= $pagination_links['prev']['url'] ?>" title="Previous page">&lt;</a>
-                        <?php }
-                        foreach ( $pagination_links['nums'] as $link ) { ?>
-                        <a href="<?= $link['url'] ?>"<?= $link['current'] ? ' class="search-result-pagination__current-page"' : '' ?>><?= $link['page'] ?></a>
-                        <?php }
-                        if ( isset($pagination_links['next']) ) { ?>
-                        <a href="<?= $pagination_links['next']['url'] ?>" title="Next page">&gt;</a>
-                        <a href="<?= $pagination_links['lastpage']['url'] ?>" title="Final page">&gt;&gt;</a>
-                        <?php } ?>
-                    </div>
-                    <?php } ?>
-                <?php } ?>
+                      <?php if ( $pagination_links ) { ?>
+                        <div class="search-result-pagination">
+                          <?php if ( isset($pagination_links['prev']) ) { ?>
+                            <a href="<?= $pagination_links['firstpage']['url'] ?>" title="First page">&lt;&lt;</a>
+                            <a href="<?= $pagination_links['prev']['url'] ?>" title="Previous page">&lt;</a>
+                          <?php } ?>
+                          <?php foreach ( $pagination_links['nums'] as $link ) { ?>
```

Comment:
I had a quick look at http://mysociety.github.io/coding-standards.html and couldn't spot any definitive guidance either way. Do you have a preference? Mine would be for single-line `<?php … ?>` statements where possible. It makes it easier to follow the nesting, since you only have to look for `<` characters, rather than also considering the indentation and trailing `{` brackets inside the PHP code block. It's a bit more Jekyll-like, and it also subconsciously discourages big chunks of view logic inside the templates.

I'll push a change to get https://github.com/mysociety/theyworkforyou/pull/1037/files#diff-35055ed30e73355e6ad23b7b10c537f5R29 into the same style too, if you're happy with that?